src/styles/colors.js

```diff
@@ -1,6 +1,8 @@
 const colors = {
   alabaster: '#fafafa',
   blue: '#006aff',
+  boxshadowgrey: '#3377ff1f',
+  darkshadow: '#22222229',
```

Comment:
It is recommended to stick to one naming convention, in this case a camelCase 🙏 :
```suggestion
  boxShadowGrey: '#3377ff1f',
  darkShadow: '#22222229',
```